cmd/xc/usage.txt

```diff
@@ -1,12 +1,13 @@
-xc [task] [inputs...]
+xc <task> [inputs...]
   Run a task from an xc compatible markdown file.
   If -file is not specified and no README.md is found in the current directory,
     xc will search in parent files for convenience.
-  -f -file string
-        Specify a markdown file that contains tasks.
-        Default: "README.md"
+  -f -file <string>
+        Specify a markdown file that contains tasks (default "README.md").
   -md
         Print the markdown for a task rather than running it.
+  -H -heading <string>
+        Specify the heading for xc tasks. (default "tasks")
```

Comment:
Since the actual heading is lowercased before comparison in `NewParser()`, and since headings are typically capitalized, wouldn't it make sense to document the default as "Tasks" instead in this help text? Otherwise it may be slightly confusing to people as it suggests that the heading must be lowercase.

Reply:
For future reference, this was changed in #51. Thanks for the quick fix, @joerdav!